index.js

```diff
@@ -13,6 +13,36 @@ const CardModal = ReadElement('.modal-section');
 const emailer = ReadElement('#useremail');
 const validator = ReadElement('.validator');
 const submitbtn = ReadElement('#submit');
+const username = ReadElement('#username');
+const message = ReadElement('#message');
+
+let myMessage;
+let userName;
+let email;
+
+let userData = [];
+
+function UpdateInput(selected) {
+  function myFunction() {
+    email = emailer.value;
+    userName = username.value;
+    myMessage = message.value;
+    userData = [userName, email, myMessage];
+    localStorage.setItem('userData', userData);
+  }
+  selected.addEventListener('change', myFunction);
+}
+
+myMessage = UpdateInput(message);
+userName = UpdateInput(username);
+email = UpdateInput(emailer);
+
+const myFormData = localStorage.getItem('userData');
+const myFormDataArray = myFormData.split(',');
+
+if (myFormDataArray.length > 0) {
+  [username.value, emailer.value, message.value] = myFormDataArray;
```

Comment:
@BertrandConxy hello and thanks for the review, but I think that I already have an Event Listener on all of the inputs.
because I use UpdateInput() on all of them, which has Event Listener

Reply:
check line 33

Reply:
@Bellagirl-maker

Reply:
@BertrandConxy